src/software/backend/radio_backend.h

```diff
@@ -11,11 +11,11 @@ class RadioBackend : public Backend
 
     RadioBackend();
 
+    void onValueReceived(ConstPrimitiveVectorPtr primitives) override;
+
```

Comment:
Changes to this file need to be reverted after final tests are performed (after any comments are addressed).